readme.md

```diff
@@ -1,21 +1,76 @@
 # Tails: chasing comets with the Zwicky Transient Facility and Deep Learning
```

Comment:
May be "using Deep Learning"? Right now ZTF and DL could be construed to be on-par this way.

Reply:
That's the intent! I'm foreseeing adding transfer learning examples for other surveys.